src/Sylius/Bundle/CurrencyBundle/Templating/Helper/CurrencyHelper.php

```diff
@@ -58,13 +62,7 @@ public function convertAmount($amount, $currency = null)
     }
 
     /**
-     * Convert amount and format it!
-     *
-     * @param int         $amount
-     * @param string|null $currency
-     * @param bool        $decimal
-     *
-     * @return string
+     * {@inheritdoc}
```

Comment:
these changes look good but watch out for including unrelated changes in PRs, should be fine though as long as there's not many